src/CLI/test.ts

```diff
@@ -14,7 +14,7 @@ import { formatDiagnostics } from "Shared/util/formatDiagnostics";
 import { getRootDirs } from "Shared/util/getRootDirs";
 import { isPathDescendantOf } from "Shared/util/isPathDescendantOf";
 
-const DIAGNOSTIC_TEST_NAME_REGEX = /^(\w+)(?:\.\d+)?$/;
+const DIAGNOSTIC_TEST_NAME_REGEX = /^(\w+)(?:\.\d+)?\.?(\d+)?$/;
```

Comment:
So the pattern here is `diagnostic.1.1.ts`? That gets a bit confusing.

Reply:
It's now unnecessary, but it was added so you could do `noAny.5.2.ts` to make a test want 2 diagnostics instead of exactly one.

Reply:
I ended up not needing that 2-diagnostic test, so this change could be reverted now. I think it might be useful later thigh.

Reply:
I wonder if in-file comment annotations would be better than trying to fit a schema into the file name. Also for multiple test cases for a single diagnostic, maybe we should support a folder with the name of that diagnostic + sub files for each test case.

Reply:
It's something to think about, I'll open a new issue and remove the change for now.